docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,265 @@
 # Getting Started with F# in Visual Studio Code with Ionide
 
-> [!NOTE]
-This is still in-progress.
\ No newline at end of file
+You can write F# in [Visual Studio Code](https://code.visualstudio.com) with the [Ionide plugin](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp), to get a cross-platform, lightweight IDE experience with IntelliSense and basic code refactorings.
```

Comment:
Probably, you could add somewhere link also to ionide.io. "For more information visit ionide.io" or something like that.